src/pages/Market/MarketCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Card } from 'components';
+import { cn } from 'utilities';
+
+import { Stat } from '../types';
+
+export interface Legend {
+  label: string;
+  color: 'blue' | 'red' | 'green';
+}
+
+export interface MarketCardProps {
+  title: string;
+  legends?: Legend[];
+  stats?: Stat[];
+  className?: string;
+  children?: React.ReactNode;
+  testId?: string;
+}
+
+export const MarketCard: React.FC<MarketCardProps> = ({
+  children,
+  title,
+  legends = [],
+  stats = [],
+  className,
+  testId,
+}) => (
+  <Card className={cn('space-y-6', className)} data-testid={testId}>
+    <div className="md:flex md:items-center md:justify-between">
```

Comment:
On the designs for the market page, all the breakpoints seem to follow this style, not just md and up